theano/tensor/tests/test_opt.py

```diff
@@ -3446,6 +3446,8 @@ def test_local_fill_useless():
     f = function([x], T.fill(x, x) * 2, mode=mode_opt)
     assert [node.op for node in f.maker.fgraph.toposort()] == [T.mul]
     f(x_)
+    # Julian: This doesn't work. See comments inside local_fill_cut.
+    # assert check_stack_trace(f, ops_to_check='all')
```

Comment:
OK. We would have to identify which other optimization is at play here, and I did not manage to do that quickly.
Let's put a placeholder comment.